.github/workflows/build.yml

```
@@ -284,7 +345,7 @@ jobs:
       run: |
         export SHELL=''
         set +e
-        go test -timeout 10m github.com/ActiveState/cli/test/integration -test.v -test.run "^TestLanguageMsiActivePerl|TestDeployIntegrationTestSuite|TestInitIntegrationTestSuite|TestPushIntegrationTestSuite|TestUpdateIntegrationTestSuite|PrepareIntegrationTestSuite|InstallScriptsIntegrationTestSuite|ShowIntegrationTestSuite|PerformanceIntegrationTestSuite$"
+        go test -timeout 10m `go list ./... | grep integration` -test.v -test.run "^TestLanguageMsiActivePerl|TestDeployIntegrationTestSuite|TestInitIntegrationTestSuite|TestPushIntegrationTestSuite|TestUpdateIntegrationTestSuite|PrepareIntegrationTestSuite|InstallScriptsIntegrationTestSuite|ShowIntegrationTestSuite|PerformanceIntegrationTestSuite$"
```

Just a suggestion:
```suggestion
        TEST_SUITE_TAGS=critical:msi:deploy:init:push:update:prepare:installer:install-scripts:show:performance go test -timeout 10m `go list ./... | grep integration` -test.v
```

If we move to that scheme, we can become a little bit more selective in the future by adding only selected tests from a files to the critical path.

---

I like this approach, but I don't want to complicate this PR any further with potential fallout from an unrelated change.

https://www.pivotaltracker.com/story/show/179693154